one_fm/overrides/leave_application.py

```diff
@@ -644,29 +644,33 @@ def reassign_to_applicant(employee: str, leave_name: str):
 
 @frappe.whitelist()
 def send_proposed_date_email(doc_name):
-    doc = frappe.get_doc("Leave Application", doc_name)
     frappe.db.set_value("Leave Application",doc_name,'workflow_state',"New Dates Proposed") 
```

Comment:
If you are going to do this, you can also use the apply workflow method, just in case there are assignment rules or stuffs like that

Reply:
yes right now , the proposed dates are implemented independently because of this its implemented in this way . to make the proposed dates in workflow its another chore that i am currently working on , i shall remove this update once on doing that .